kpops/component_handlers/kafka_connect/kafka_connect_handler.py

```diff
@@ -120,9 +120,7 @@ def __dry_run_connector_creation(
                 f"Connector Creation: validating the connector config for connector {connector_name} resulted in the following errors:"
             )
             log.error("\n".join(errors))
-            sys.exit(
-                1
-            )  # FIXME raise instead https://github.com/bakdata/kpops/issues/101
+            raise ValueError(f"Error creating connector {connector_name}")
```

Comment:
Don't log and throw
https://stackoverflow.com/questions/6639963/why-is-log-and-throw-considered-an-anti-pattern